src/core/src/bootstrap/Constants.py

```diff
@@ -208,7 +208,8 @@ class AutoAssessmentStates(EnumBackport):
     MAX_INSTALLATION_RETRY_COUNT = 3
     MAX_IMDS_CONNECTION_RETRY_COUNT = 5
     MAX_ZYPPER_REPO_REFRESH_RETRY_COUNT = 5
-    MAX_BATCH_SIZE_FOR_PACKAGES = 3
+    MAX_BATCH_SIZE_FOR_PACKAGES = 6
+    NUMBER_OF_PACKAGES_IN_BATCH_COULD_TAKE_MAX_TIME_TO_INSTALL = 3
```

Comment:
Not blocking PR on this as I'm not able to suggest anything better off the top of my head, but this variable name not self-explanatory even when you know what its purpose really is. It'll only get more confusing for people who may come later, and even for us as the context of this PR fades.

If any of you think of a better name later, please raise -- this must be improved on. @rane-rajasi @GAURAVRAMRAKHYANI